README.md

```diff
@@ -74,13 +71,265 @@ storyblokInit({
+  onMount(async () => {
+    const storyblokApi = getStoryblokApi();
+    const { data } = await storyblokApi.get("cdn/stories/home", {
+      version: "draft",
+    });
+    story = data.story;
+    useStoryblokBridge(story.id, (newStory) => (story = newStory));
+  });
+</script>
+```
+
+You can pass [Bridge options](https://www.storyblok.com/docs/Guides/storyblok-latest-js?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-svelte) as a third parameter as well:
+
+```js
+useStoryblokBridge(story.id, (newStory) => (story = newStory), {
+  resolveRelations: ["Article.author"],
+});
+```
+
+### 3. Link your components to Storyblok Visual Editor
+
+To link the Storyblok components, you have to
+
+- Load them in components when calling `storyblokInit`
+
+- Use the `storyblokEditable` action on the root element of each component
+
+```html
+<div use:storyblokEditable={blok} / >
+```
+
+- Use the `StoryblokComponent` to load them by passing the `blok` property
+
+```html
+<StoryblokComponent {blok} />
+```
+
+> The `blok` is the actual blok data coming from [Storblok's Content Delivery API](https://www.storyblok.com/docs/api/content-delivery/v2?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-svelte).
+
+### Features and API
+
+You can **choose the features to use** when you initialize the plugin. In that way, you can improve Web Performance by optimizing your page load and saving some bytes.
+
+#### Storyblok API
+
+You can use an `apiOptions` object. This is passed down to the [storyblok-js-client config object](https://github.com/storyblok/storyblok-js-client#class-storyblok):
+
+```js
+storyblokInit({
+  accessToken: "<your-token>",
+  apiOptions: {
+    //storyblok-js-client config object
+    cache: { type: "memory" },
+  },
+  use: [apiPlugin],
+});
+```
+
+If you prefer to use your own fetch method, just remove the `apiPlugin` and `storyblok-js-client` won't be added to your application. You can find out more about our [Content Delivery API](https://www.storyblok.com/docs/api/content-delivery?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-svelte) in the documentation.
+
+#### Storyblok Bridge
+
+You can conditionally load it by using the `bridge` option. Very useful if you want to disable it in production:
+
+```js
+storyblokInit({
+  bridge: process.env.NODE_ENV !== "production",
+});
+```
+
+Keep in mind you have still access to the raw `window.StoryblokBridge`:
+
+```js
+const sbBridge = new window.StoryblokBridge(options);
+sbBridge.on(["input", "published", "change"], (event) => {
+  // ...
+});
+```
+
+For background information on the [Storyblok JS Bridge](https://www.storyblok.com/docs/Guides/storyblok-latest-js?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-svelte), please check out the documentation.
+
+### Rendering Rich Text
+
+You can easily render rich text by using the `renderRichText`function that comes with `@storyblok/svelte`and Sveltes `{@html htmlstring}`directive.
```

Comment:
There are spaces missing after "renderRichText", "@storyblok/svelte" and "htmlstring"